python/enthic/compute_stats.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================================================
+Compute statistics from companies's data after complete and check it
+=========================================================================
+
+PROGRAM BY PAPIT SASU, 2019
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import json
+import math
+
+def convert_data_to_tree(rawData):
+    """
+    The def convert_data_to_tree takes an empty CompteDeResultat tree
+    from "arbre_compte_de_resultat.json" file, and fill it with data from enthic API
+    """
+    # Opening JSON file
+    json_file = open('arbre_compte_de_resultat.json',)
+    new_tree = json.load(json_file)
+    recursive_fill_tree(new_tree, rawData)
+    return new_tree
+
+def recursive_fill_tree(itemToFill, rawData):
+    # Begin to fill children if there is children
+    if 'children' in itemToFill:
+        for child_name in itemToFill["children"]:
+            recursive_fill_tree(itemToFill["children"][child_name], rawData)
+
+    # Find corresponding value from rawData
+    i = 0
+    while i < len(rawData):
+        for code in rawData[i]:
+            if code in itemToFill["codeLiasses"]:
+                itemToFill['data'] = rawData[i][code]
+                itemToFill['data']["status"] = "official"
+                itemToFill['data']['code'] = code
+                del rawData[i]
+                break
+        i = i + 1
+
+def check_tree_data(item):
```

Comment:
What is the pupose of that function, it does not return a thing, does not raise exception, the only side effect is a print. Replace it by a log if it is the only desired effect please.